sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 4.4.0-beta.2 (Unreleased)
 
+### Features Added
+- Added `Exportable` and `ReleasePolicy` to `CreateKeyOptions`, `ImportKeyOptions`, and `KeyProperties` to support Secure Key Release for Key Vault and Managed HSM.
```

Comment:
Have you been able to get SKR working on AKV? It works on HSM, but when I try to create an exportable key -- on a premium SKU vault -- I get back this exception:
`(BadParameter) AKV.SKR.1001: The exportable attribute is only supported with premium vaults and API version >= '7.3-preview'.`
(This may be something to bring up with the service team, since my request met both the error response's criteria and still failed.)

Reply:
I think @maorleger or @heaths brought this up last week. I am currently writing some tests but I suspect I will see the same error message. I'll let you know what I actually get back.

Reply:
Talked about this directly with @mccoyp, but I'm putting it here for the record. Azure Key vault does not support releasing keys as of this moment but it's in the roadmap. Support for this feature is only available for Managed HSMs.